framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/deployment/KieServerDeploymentImpl.java

```diff
@@ -36,21 +36,19 @@
 
 
     @Override public URL getUrl() {
+        if (url == null) {
+            url = getHttpRouteUrl(serviceName);
```

Comment:
What if we set URL for Kie server later, that doesn't match to created URL using default subdomain?

Reply:
You mean the case when we need to know the URL in advance and we know that it will be different from default subdomain?
It that case we could reintroduce setUrl method where we can specify the URL. I deleted it because I don't expect that we will cover such scenario and I want to keep the code as simple as possible.

Reply:
+1 for simple code.